Orso.Arpa.Application/Services/AuditlogService.cs

```diff
@@ -22,7 +22,7 @@ public AuditLogService(IMediator mediator, IMapper mapper)
             _mediator = mediator;
         }
 
-        public async Task<IEnumerable<AuditLogDto>> GetAsync(Guid entityId, int? skip, int? take)
+        public async Task<IEnumerable<AuditLogDto>> GetAsync(Guid? entityId, int? skip, int? take)
         {
             IQueryable<AuditLog> entities = await _mediator.Send(new List.Query<AuditLog>(
                 predicate: v => v.KeyValues.Contains(entityId.ToString()),
```

Comment:
wenn entityId null ist, müsste doch entityId.ToString() eine exception werfen? Muss nicht das predicate entfernt werden, wenn entityid null ist? oder wird diese methode gar nicht aufgerufen in dem fall?

Reply:
das hatte ich auch zunächst erwartet und habe es (dank test-driven) dann plausibilisiert. Offensichtlich kommt die Funktion mit dem nullable Datentyp zurecht.